styles.css

```
@@ -0,0 +1,290 @@
+*{
+  font-family: 'GTWalsheim','Didact Gothic' , sans-serif;
+}
+
+body{
+  margin: 0;
+}
+
+/*--------------------------------seccion NAV---------------------------------*/
+
+
+.mainNav{
+  display: flex;
+  justify-content: space-between;
+  margin: 26px 32px;
+}
+
+.closed.menu-items-toggle, .closed .home-logo,.closed .phone-link img{
+  filter: grayscale(0) brightness(0);
+}
+
+.home-logo .logoBig{
+  display: none;
+}
+
+/*------phone tooltip-----*/
+.phone-link{
+    position: relative;
```

Use 2-space indentation 
```suggestion
  position: relative;
```